lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -206,7 +206,27 @@ for node in $allocated_nodes; do
 done
 
 # Make sure all openstack services are down on all nodes before we stop rabbitmq
-sleep 10
+services_stopped=""
+count=0
+while [ "x$services_stopped" != "xyes" -a "$count" -lt 100 ]; do
+  sleep 1
```

Comment:
Can you add a `echo "Waiting for OpenStack services to stop..."` before the `sleep`?